Sources/Core/Countries.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  Countries.swift
+//  Country Picker
+//
+//  Created by Uwe Tilemann on 07.03.24.
+//
+
+import Foundation
+
+public struct Country {
+    public static var `default`: [Country] = loadJSON("Countries")
```

Comment:
Ich habe erwartet das es ein `default` Country ist, anstatt ein Array aller Countries. Was hältst du von folgender Suggestions?

```suggestion
    public static var all: [Country] = loadJSON("Countries")
    public static var germany: Country = Country(code: "DE", label: "Germany", callingCode: 49)
```

Reply:
Die Variable `germany` könnten wir dann in der View nutzen wie `all`, so das die Definition im Struct stattfindet, anstatt in der View.

Reply:
https://github.com/snabble/snabble-phoneauth-ios-sdk/pull/12/commits/f9ea062f4f0c9549cfee688cbb0def000d263f7f